synonymdev/bitkit-react-native
# feat(ui): Update small copy for consistency

### Description

Implements small copy changes to increase consistency, **using design v45 as reference**:
- 1️⃣ **Payment Received Event:**
  Title changed to 'Received Bitcoin' (on-chain) and 'Received Instant Bitcoin' (Lightning) for consistency.
- 2️⃣ **Recovery Phrase Back up Successful:**
  Enhanced emphasis by highlighting 'secure place' and 'only way to recover' parts in bold, white text.
- 3️⃣ **LN onboarding > Choose package:**
  Updated body copy for consistency: 'Choose how much bitcoin you want to spend instantly.'
- 4️⃣ **LN onboarding > Custom amount:**
  Updated body copy for consistency: 'Enter the amount of bitcoin you want to spend instantly.'

### Linked Issues/Tasks

Subtasks 5-8 from [Bitkit Mobile Prototype v45](https://app.asana.com/0/inbox/1206229374951486/1206354352883293/1206354201869938/f)

### Type of change


### Tests


### Screenshot / Video

| 1️⃣ | 2️⃣ | 3️⃣ | 4️⃣ |
| - | - | - | - |
| <img width="200" src="https://github.com/synonymdev/bitkit/assets/4588074/2b3345d8-afe8-4ebd-8c3e-7131377a6828"> | <img width="200" src="https://github.com/synonymdev/bitkit/assets/4588074/afe40a2b-74e3-4eef-a9fb-5a0b18248f10"> | <img width="200" src="https://github.com/synonymdev/bitkit/assets/4588074/ae60163e-c7fe-4b8e-9f06-57efb0f3dd3f"> | <img width="200" src="https://github.com/synonymdev/bitkit/assets/4588074/e42924d1-1e9c-448c-8249-a1d2acea1aff"> |

### QA Notes

See above screengrabs.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206354352883299
  - https://app.asana.com/0/0/1206354352883298
  - https://app.asana.com/0/0/1206354352883301
